src/dragonball/src/dbs_pci/src/lib.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2023 Alibaba Cloud. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2018 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE-BSD-3-Clause file.
+//![deny(missing_docs)]
+//!
+//! Implements PCI devices and buses.
+//!
+//! The role and relationship about PCI related traits/structs:
+//! - PCI root: a pseudo device to handle PCI configuration accesses.
+//! - PCI bus: a container object to hold PCI devices and resources, corresponding to the PCI bus
+//!   defined in PCI/PCIe specs.
+//! - PCI root bus: a special PCI bus which has no parent PCI bus. The device 0 under PCI root bus
+//!   represent the root bus itself.
+//! - PCI device: the real object to emulate a PCI device. For most PCI devices, it needs to
+//!   handle accesses to PCI configuration space and PCI BARs.
+//! - PCI configuration: a common framework to emulator PCI configuration space header.
+//! - PCI MSI/MSIx: structs to emulate PCI MSI/MSIx capabilities.
+
+use std::sync::Arc;
+
+use dbs_device::device_manager::IoManagerContext;
+use dbs_interrupt::KvmIrqManager;
+
+mod bus;
```

Comment:
move mod ahead of private use statement?

Reply:
done